RELEASE_PROCESS.md

```diff
@@ -1,14 +1,15 @@
-# TektonCD CLI release process
+# TektonCD CLI Release Process
 
-- You need to be a member of tektoncd-cli [OWNERS](OWNERS) to be able to push the new branch.
+- You need to be a member of tektoncd-cli [OWNERS](OWNERS) and be a member of the [CLI maintainers team](https://github.com/orgs/tektoncd/teams/cli-maintainers) to be able to push a new branch for the release. You will also need to be a member of the [Homebrew maintainers team](https://github.com/orgs/tektoncd/teams/homebrew-maintainers) to be able to update the CLI Homebrew formula.
```

Comment:
Note that `CLI maintainers team` member are now automatically maintainer on `homebrew-tools` (for the formula), it make it simpler to manage :angel: 
`homebrew-mantainers` are admins of the `homebrew-tools` repository.

Reply:
Ah, sounds good! I'll make an edit.